pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -134,7 +136,7 @@ func GetOpenshiftBootstrapClusterRoles() []rbacv1.ClusterRole {
 				Labels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-cluster-reader": "true"},
 			},
 			Rules: []rbacv1.PolicyRule{
-				rbacv1helpers.NewRule(read...).Groups(kapiGroup).Resources("componentstatuses", "nodes", "nodes/status", "persistentvolumeclaims/status", "persistentvolumes",
+				rbacv1helpers.NewRule(read...).Groups(kapiGroup).Resources("componentstatuses", "configmaps", "nodes", "nodes/status", "persistentvolumeclaims/status", "persistentvolumes",
```

Comment:
This looks suspicious.

Reply:
@smarterclayton mentions in the ticket, configmaps should be added.